cartography/intel/okta/test.py

```diff
@@ -0,0 +1,796 @@
+# Okta intel module
+import logging
+import json
+from okta import UsersClient, UserGroupsClient, AppInstanceClient, FactorsClient
+from okta.framework.PagedResults import PagedResults
+from okta.models.usergroup import UserGroup
+from okta.framework.ApiClient import ApiClient
+from okta.framework.OktaError import OktaError
+# from okta import UserGroupsClient
+# from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+
+def _get_okta_users(user_client):
+    user_list = []
+    paged_users = user_client.get_paged_users()
```

Comment:
Please refactor the data massaging (taking the returned data from `paged_users` and making adjustments and adding it to your own list) into a separate `transform` function.  Example [here](https://github.com/lyft/cartography/blob/master/cartography/intel/gcp/compute.py#L131)